server/models/get-article-by-id-model.js

```diff
@@ -1,6 +1,28 @@
 const db = require("../../db/connection.js")
+const { selectArticles } = require("./get-articles-model.js")
 
-exports.selectArticleById = (id) => {
+exports.selectArticleById = (id, commentCount) => {
+    if (commentCount === "comment_count") {
+        return selectArticles()
+        .then((articles) => {
+            const selectedArticle = articles.filter(article => {
```

Comment:
The comment count shouldn't be a query. It is extra functionality that we want you to add onto your previous endpoint. This should be achieved by altering the SQL query and tested by adding just one test to your previous tests (and ensuring all the others still pass)